src/components/dashboard-widgets/dashboard-text-widget/dashboard-text-widget.component.ts

```diff
@@ -0,0 +1,83 @@
+import {ChangeDetectionStrategy, Component, ElementRef, EventEmitter, Input, Output, ViewChild} from '@angular/core';
```

Comment:
I notice the formatting on imports doesn't match our other components which all use spaces between the braces and the identifiers